components/script/dom/htmldocument.rs

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use std::ptr::NonNull;
+
+use dom_struct::dom_struct;
+use js::jsapi::JSObject;
+
+use crate::dom::bindings::codegen::Bindings::HTMLDocumentBinding::HTMLDocumentMethods;
+use crate::dom::bindings::error::{Error, Fallible};
+use crate::dom::bindings::reflector::Reflector;
+use crate::dom::bindings::root::DomRoot;
+use crate::dom::bindings::str::DOMString;
+use crate::dom::location::Location;
+use crate::script_runtime::JSContext as SafeJSContext;
+
+// https://html.spec.whatwg.org/multipage/#htmldocument
```

Comment:
can you follow other doc link and use this pattern: `/// <link>` for all doc link in this PR?